packages/manager/apps/web/client/app/hosting/website-coach/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,12 @@
+{
+  "hosting_website_coach_title": "Website Coach",
+  "hosting_website_coach_button": "Activer Website Coach",
+  "hosting_website_coach_1dn_text1": "Recevez régulièrement des conseils basés sur une analyse à 360° de votre site",
+  "hosting_website_coach_1dn_text2": "sur l'adresse email",
+  "hosting_website_coach_1dn_text3": "pour en améliorer votre audience, la sécurité et plus généralement la satisfaction de vos visiteurs.",
```

Comment:
I suppose this is due to some markup being added but it might be difficult to translate

Reply:
Yes, we a `strong` and a `a` between each

Reply:
I've updated to a single translations with values